### DISCUSSION TITLE
How can signserver CE support LTV-enabled when signed pdf-file being opened with Adobe after signing with PDFSigner?

### DISCUSSION THREAD

**Discussion by tommy-inteluck** [Keyfactor/signserver-ce#77] [Q&A]
I have arleady used CA cetificate with AATL and configured TSA_URL when signing with PDFSigner. 
After that, when I opened the file with Adobe, and the signature can pass AATL validation and secure timestamp validation. 
But still showed 'LTV not enabled'.

Can signserver CE support that. If so , what am I missing while signing whith PDFSigner?

**Comment by mlundblad** [comment]
I think for the PDF to be considered LTV-enabled, the signature needs to include revocation information. I.e. the signing certificate needs to include revocation information (either a URL for an OCSP responder providing revocation lookup for the issuing CA, or a link to a CRL download for revocation list).

**Reply by tommy-inteluck** [reply]
Thankss
How the signing certificate include revocation information as you mentioned?  
We bought the certificate from a CA within AATL.  And I have tried the overridable parameters of PDFSigner when signing, like set EMBED_CRL=True and EMBED_OCSP_RESPONSE=True, but still cannot enble LTV. Is this way I used the method you mentioned?

Looking forward to your reply

**Reply by nicoveiguela** [reply]
Hi Tommy, 

I was wondering where did you buy the CA. Im building a app on top of signserver and im looking for a CA thats trusted by adobe. Thanks in advance,

**Reply by AkshayD20** [reply]
Hi nicoveiguela,
You need to include full certificate chain into with ROOTCA CMS while signing pdf hash then adobe automatically trusted the signature.

**Reply by nicoveiguela** [reply]
Thanks Akshay! 

I was asking tommy where did he buy the root certificate cause all the options I found they come with hardware (USB) and this is not possible for us cause we use cloud servers..

**Reply by tommy-inteluck** [reply]
Hi nicoveiguela,
You can check this AATL members in this link: https://helpx.adobe.com/acrobat/kb/approved-trust-list1.html
You can buy the certificate from any provider in this member list. Maybe you would like to chose a provider in your country.

**Reply by nicoveiguela** [reply]
Thank you Tommy, 

I was wondering which one you bought and if you got it online? Im asking because it has been hard to find a certificate with out hardware device.

**Reply by tommy-inteluck** [reply]
I use WoTrus, you can check it in the CN section of AATL list. 
I don't know if it has English version website for its products.

**Reply by nicoveiguela** [reply]
Thanks tommy .
Last one for you , did you bought the soft one? If they offer the soft version I think it is what I need. 
https://www.wosign.com/Products/document_signing.htm

Thanks for your support! Regards from Argentina

**Reply by tommy-inteluck** [reply]
yes.  we use it for pdf signing. 
you can contact their sales.

**Comment by AkshayD20** [comment]
Hi Tommy,

To LTV enabled you need to timestamp signed pdf content and include into pdf signature with pdf, then Adobe can check  LTV enabled or not.
